.gitignore

```diff
@@ -0,0 +1 @@
+MCsquare/bin/Materials/
```

Comment:
In this case I think it is reasonable. The Materials Directory of MCsquare should never be committed, because we provide the materials als zip file which get automatically unzipped into the materials directory (its a lot of files). So in this case we want to make sure this folder is never in the repo also to have a clean history.